py-polars/tests/unit/io/test_lazy_ipc.py

```diff
@@ -85,3 +86,27 @@ def test_ipc_list_arg(io_files_path: Path) -> None:
     assert df.shape == (54, 4)
     assert df.row(-1) == ("seafood", 194, 12.0, 1)
     assert df.row(0) == ("vegetables", 45, 0.5, 2)
+
+
+def test_scan_ipc_local_with_async(
+    capfd: Any,
+    monkeypatch: Any,
+    io_files_path: Path,
+) -> None:
+    monkeypatch.setenv("POLARS_VERBOSE", "1")
+    monkeypatch.setenv("POLARS_FORCE_ASYNC", "1")
+
+    assert_frame_equal(
+        pl.scan_ipc(io_files_path / "foods1.ipc").head(1).collect(),
+        pl.DataFrame(
+            {
+                "category": ["vegetables"],
+                "calories": [45],
+                "fats_g": [0.5],
+                "sugars_g": [2],
+            }
+        ),
+    )
+
+    captured = capfd.readouterr().err
+    assert "ASYNC READING FORCED" in captured
```

Comment:
There is a test.

Reply:
But it fails on windows with:

```
polars.exceptions.ComputeError: cannot parse bucket (host) from url: d:\a\polars\polars\py-polars\tests\unit\io\files\foods1.ipc
```

It seems like `object_store::path::Path::parse` does not like windows-style paths. We could try converting windows to unix style and then passing it to parse, or ignoring the test like so:

```py
@pytest.mark.skipif(
    sys.platform =="win32", reason="object_store does not handle windows-style paths."
)
```

@ritchie46 what do you think we should do?

Reply:
Skip the test.